nifi-toolkit/nifi-toolkit-api/pom.xml

```diff
@@ -83,7 +84,7 @@ language governing permissions and limitations under the License. -->
             <plugin>
                 <groupId>io.swagger</groupId>
                 <artifactId>swagger-codegen-maven-plugin</artifactId>
-                <version>2.2.2</version>
+                <version>2.4.27</version>
```

Comment:
It is unclear if moving to swagger 3 would help this issue (i.e. using a more current version of okhttp) as it appears it still uses okhttp 2.  See https://github.com/swagger-api/swagger-codegen/blob/3.0.0/samples/client/petstore/java/okhttp-gson/pom.xml#L215

Reply:
Thanks for the note @NissimShiman! Moving to Swagger 3 is a good goal, but as you noted, it appears that the codegen plugin still depends on an old version of OkHttp.